### ISSUE TITLE
[Feature Request] Abillity to set static room name/url for specific channel

### ISSUE THREAD

**Issue by Centzilius** [mattermost/mattermost-plugin-jitsi#134]
Hi, would it be possible to specify a specific jitsi url / meeting name for a channel?

**Comment by jespino** [comment]
That would be a new feature, what is the problem that you want to solve? What is the use case?

**Comment by Centzilius** [comment]
We have a channel for our regular (online) meetups and the jitsi link is already distributed and always stays the same (we marked the jitsi conference as permanent on the jitsi internal jabber server). It would be nice to be able to click on the conference button in the corresponding mattermost channel so that everyone knows that the meetup has begun.

**Comment by jespino** [comment]
@Centzilius Yes, that makes a lot of sense, actually I was thinking about the concept of schedule meeting that would work similar to that, except for the fact that you don't need to click the button at all. My point with that is, if is a recurrent meeting, that always use the same URL, should be published through some kind of "scheduler".

My concern is about publishing always a meeting for the channel, and if somebody needs an "ad-hoc" meeting for that channel would need to use other way to create it (for example `/jitsi my-ad-hoc-meeting`).

Other interesting feature that I'm thinking about is, auto-capture internal links to your Jitsi server and open them in the embedded jitsi (if you have embedded enabled). That would allow you to add the Jitsi Meeting link in the header of the channel.

**Comment by jespino** [comment]
@jasonblais some interesting ideas here and a legitimate use case that we have to decide how to address :point_up:

**Comment by jasonblais** [comment]
cc @aaronrothschild ^

**Comment by sigboe** [comment]
@jespino What about adding an other button next to the `Start Jitsi Meeting` button, so you have one for `Start Jitsi Meeting` and one for `Start Ad-hoc Jitsi Meeting`. I would suggest the current icon being used for the new feature, and the current functionality getting a new icon with maybe something like a star in it? Or the other way around.

We also use static meeting room names for different functions

> My point with that is, if is a recurrent meeting, that always use the same URL, should be published through some kind of "scheduler".

Yes we would use this feature if you added the ability to also add a random password every time it is shedueled to start, so that people with access to the mattermost room always have "This occurrence's" password.

**Comment by Rintan** [comment]
Are there any progress of static room name?
